tools/cgreen-runner.c

```diff
@@ -32,16 +32,25 @@ static void version(void) {
 /*----------------------------------------------------------------------*/
 static void usage(const char *program_name) {
     printf("cgreen-runner for Cgreen unittest and mocking framework v%s\n\n", VERSION);
-    printf("Usage:\n    %s [--xml <prefix>] [--suite <name>] [--verbose] [--quiet] [--no-run] [--help] (<library> [<test>])+\n\n", program_name);
+    printf("Usage:\n    %s [--suite <name>] [--verbose] [--quiet] [--no-run] "
+#if HAVE_XML_REPORTER
+           "--xml <prefix>"
```

Comment:
You need a space at the end here, as well as in the --libxml2 line.

Reply:
Fixed.